persistence-mysql/src/main/resources/flux_scheduler/migrations/0003_create_ClientElb_table.sql

```diff
@@ -0,0 +1,17 @@
+--liquibase formatted sql
+
+--changeset akif.khan:3 runOnChange:false
+
+CREATE TABLE IF NOT EXISTS `ClientElb` (
+  `id` VARCHAR(64) NOT NULL,
+  `elbUrl` VARCHAR(300) NOT NULL,
+  `createdAt` TIMESTAMP(3) DEFAULT CURRENT_TIMESTAMP(3),
+  `updatedAt` TIMESTAMP(3) DEFAULT CURRENT_TIMESTAMP(3) ON UPDATE CURRENT_TIMESTAMP(3),
```

Comment:
`updatedAt` TIMESTAMP(3) NOT NULL DEFAULT CURRENT_TIMESTAMP(3) ON UPDATE CURRENT_TIMESTAMP(3),

Reply:
Made this Timestamp field as NOT NULL.